src/app/qgsrasterlayerproperties.cpp

```diff
@@ -1762,14 +1762,10 @@ void QgsRasterLayerProperties::loadStyle_clicked()
                        this,
                        tr( "Load layer properties from style file" ),
                        lastUsedDir,
-                       tr( "QGIS Layer Style File" ) + " (*.qml)" );
+                       tr( "QGIS Layer Properties File" ) + " (*.qlp);;" + tr( "QGIS Layer Style File" ) + " (*.qml)" );
```

Comment:
Should we rephrase this to "QGIS Layer Style File (deprecated)" to reduce the confusion about "what's the difference between the two?"?
Is the double `;;` on purpose?